test/fakeintake/aggregator/logAggregator.go

```diff
@@ -20,8 +20,8 @@ type Log struct {
 	Timestamp     int      `json:"timestamp"`
 	HostName      string   `json:"hostname"`
 	Service       string   `json:"service"`
-	Source        string   `json:"source"`
-	Tags          []string `json:"tags"`
+	Source        string   `json:"ddsource"`
+	Tags          []string `json:"ddtags"`
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Thank you !! Will add tests

Reply:
There was a test, format might have changed in between versions. 

https://github.com/DataDog/datadog-agent/blob/main/test/fakeintake/aggregator/logAggregator_test.go#L40-L42

Will update the raw payload, or we might generate payloads from the agent code and generate it with a special flag to create fixtures.